src/utils/nameUtils.ts

```diff
@@ -83,11 +82,11 @@ function getNameParts(name: string) {
 
 export function getOperationFullName(
   operationGroup: OperationGroup,
-  operation: Operation
+  operation: Operation,
+  clientName: string
 ) {
   const groupName = normalizeName(
-    getLanguageMetadata(operationGroup.language).name ||
-      TOPLEVEL_OPERATIONGROUP,
```

Comment:
Is this constant still needed, or can it be removed from the constants file?

Reply:
Yeah we can remove it from the file, good catch!